greynoise/actions/workflows/ip_lookup.yaml

```
@@ -0,0 +1,58 @@
+---
+
+version: 1.0
+
+description: >-
+  A multi-lookup workflow to provide RIOT or Context data for known IPs in GreyNoise.
+
+input:
+  - ip
+
+vars:
+  - is_riot: false
+  - is_seen: false
+  - result: ""
+
+output:
+  - is_riot: <% ctx().is_riot %>
+  - is_seen: <% ctx().is_seen %>
+  - result: <% ctx().result %>
+
+tasks:
+  check_riot:
+    action: greynoise.riot_ip
+    input:
+      ip: <% ctx().ip %>
+    next:
+      - when: <% succeeded() and result().result.riot = true %>
+        publish:
+          result=<% result().result %>
+          is_riot=true
```

These are dictionary keys with no corresponding values, which isn't valid YAML.

I believe the correct YAML and Orquesta syntax is to make them elements of a list.
```suggestion
          - result=<% result().result %>
          - is_riot=true
```

---

@blag : So, this is working before this suggested change (the workflow outputs what I expect) but when I tried making this change,  the workflow fails with:

```
{
  "errors": [
    {
      "expression": "<% succeeded() and results()|length >=1 and result().result[0].noise = true %>",
      "message": "Lexical error: illegal character '|' at position 25",
      "spec_path": "tasks.check_quick.next[0].when",
      "schema_path": "properties.tasks.patternProperties.^\\w+$.properties.next.items.properties.when",
      "language": "yaql",
      "type": "expression"
    },
    {
      "expression": "<% succeeded() and results()|length >=1 and result().result[0].noise = false %>",
      "message": "Lexical error: illegal character '|' at position 25",
      "spec_path": "tasks.check_quick.next[1].when",
      "schema_path": "properties.tasks.patternProperties.^\\w+$.properties.next.items.properties.when",
      "language": "yaql",
      "type": "expression"
    },
    {
      "message": "['result=<% result().result[0] %>'] is not valid under any of the given schemas",
      "spec_path": "tasks.check_quick.next[1].publish",
      "schema_path": "properties.tasks.patternProperties.^\\w+$.properties.next.items.properties.publish.oneOf",
      "type": "syntax"
    },
    {
      "message": "['result=<% result().result %>', 'is_riot=true'] is not valid under any of the given schemas",
      "spec_path": "tasks.check_riot.next[0].publish",
      "schema_path": "properties.tasks.patternProperties.^\\w+$.properties.next.items.properties.publish.oneOf",
      "type": "syntax"
    },
    {
      "message": "['result=<% result().stderr %>'] is not valid under any of the given schemas",
      "spec_path": "tasks.check_riot.next[2].publish",
      "schema_path": "properties.tasks.patternProperties.^\\w+$.properties.next.items.properties.publish.oneOf",
      "type": "syntax"
    },
    {
      "message": "['result=<% result().result %>'] is not valid under any of the given schemas",
      "spec_path": "tasks.check_riot.next[3].publish",
      "schema_path": "properties.tasks.patternProperties.^\\w+$.properties.next.items.properties.publish.oneOf",
      "type": "syntax"
    },
    {
      "message": "['is_riot=false', 'is_seen=true', 'result=<% result().result %>'] is not valid under any of the given schemas",
      "spec_path": "tasks.get_context.next[0].publish",
      "schema_path": "properties.tasks.patternProperties.^\\w+$.properties.next.items.properties.publish.oneOf",
      "type": "syntax"
    }
  ],
  "output": null
}
```

---

@blag : I'm confused, this exact SYNTAX is what is in the current version?  And during my testing, it works as expected?